recipe/meta.yaml

```diff
@@ -23,10 +21,10 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
     - wheel
+    - setuptools >=62.6
     - versioneer 0.29
-    - tomli
+    - toml
```

Comment:
`toml` seems unused, `versioneer` is using `tomli`: https://github.com/python-versioneer/python-versioneer/blob/0.29/pyproject.toml#L37

Reply:
Yes, it should be `tomli`, not `toml`